README.md

```diff
@@ -157,6 +157,11 @@ print(explanation$dt)
 
 # Finally we plot the resulting explanations
 plot(explanation)
+#> Registered S3 methods overwritten by 'ggplot2':
```

Comment:
Regenerate the file and see if this change gets removed? I tested locally and it was removed. Let me know if I should push that.

Reply:
Fixed in 14a9f67